numpy/core/src/umath/override.c

```diff
@@ -180,18 +179,6 @@ normalize_reduceat_args(PyUFuncObject *ufunc, PyObject *args,
 }
 
 static int
-normalize_outer_args(PyUFuncObject *ufunc, PyObject *args,
-                    PyObject **normal_args, PyObject **normal_kwds)
-{
```

Comment:
Do we want to separate the handling of reduce and accumulate here with the latter dropping keepdims?